docs/pages/chat-apps/core-messaging/group-permissions.mdx

```diff
@@ -357,6 +357,44 @@ try await group.removeMembers(inboxIds: [inboxId])
 
 :::
 
+### Leave a group
+
+A member can remove themselves from a group by calling `leaveGroup()`. After leaving, the group becomes inactive for that member.
```

Comment:
You can not leave a group if you are the super admin of the group (by default the group creator).